synapse/storage/events.py

```diff
@@ -293,19 +366,19 @@ def _persist_event_txn(self, txn, event, context, backfilled,
                 txn, event.event_id, context.rejected
             )
 
-        for hash_alg, hash_base64 in event.hashes.items():
-            hash_bytes = decode_base64(hash_base64)
-            self._store_event_content_hash_txn(
-                txn, event.event_id, hash_alg, hash_bytes,
-            )
+        # for hash_alg, hash_base64 in event.hashes.items():
+        #     hash_bytes = decode_base64(hash_base64)
+        #     self._store_event_content_hash_txn(
+        #         txn, event.event_id, hash_alg, hash_bytes,
+        #     )
```

Comment:
If they aren't being read from I guess we should remove them. We can always add them back if they turn out to be necessary.